File: test/fixtures/output/ocaml/cohttp/query.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value" in

Client.call `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: src/targets/javascript/fetch/fixtures/query.js
const options = {method: 'GET'};

fetch('https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value', options)
  .then(res => res.json())
  .then(res => console.log(res))
  .catch(err => console.error(err));